maps/cogmap.dmm

```diff
@@ -2407,7 +2407,7 @@
 "aUo" = (/obj/table/reinforced/bar/auto,/obj/item/clipboard,/obj/item/pen,/turf/simulated/floor,/area/station/crew_quarters/courtroom)
 "aUp" = (/obj/grille/steel,/obj/disposalpipe/segment,/obj/window/auto/reinforced,/turf/simulated/floor/plating,/area/station/crew_quarters/courtroom)
 "aUq" = (/obj/machinery/door/firedoor/pyro,/obj/machinery/door/airlock/pyro/glass,/turf/simulated/floor,/area/station/crew_quarters/courtroom)
-"aUr" = (/turf/simulated/wall/auto/reinforced/supernorn,/area/station/crew_quarters/courtroom)
+"aUr" = (/obj/bookshelf/persistent,/turf/simulated/floor/specialroom/chapel{dir = 8},/area/station/chapel/main)

```

Comment:
error, you deleted a cable

Reply:
i just replaced it and pushed the update

Reply:
wait i also deleted the light 1 sec